seat_service/src/lib/seat_adjuster/seat_controller/tools/ecu-reset

```diff
@@ -52,10 +52,6 @@ done
 
 echo "### Normalizing SeatAdjust ECU on $CAN"
 
-if ! ifconfig $CAN | grep -q "UP"; then
```

Comment:
Why is this check removed?

Reply:
Because otherwise the script will fail if `ifconfig` is not available (even though `$CAN` exists and is UP).

Could perhaps be made smarter, but again, if $CAN isn't available/up, the script will fail anyway.